agent/metasploit_agent.py

```diff
@@ -28,6 +29,12 @@
 MODULE_TIMEOUT = 300
 VULNERABLE_STATUSES = ["vulnerable", "appears"]
 METASPLOIT_AGENT_KEY = b"agent_metasploit_asset"
+TOKEN_REPLACE = "REPLACEME"
+REFERENCES = {
+    "CVE": f"https://nvd.nist.gov/vuln/detail/CVE-{TOKEN_REPLACE}",
+    "CWE": f"https://cwe.mitre.org/data/definitions/{TOKEN_REPLACE}.html",
+    "EDB": f"https://www.exploit-db.com/exploits/{TOKEN_REPLACE}",
+}
```

Comment:
I don't like the way these are replaced. I think you can use named format strings to replace the values directly.